packages/documentation/.storybook/main.ts

```diff
@@ -49,6 +50,13 @@ const config: StorybookConfig = {
     STORYBOOK_GTM_PAGE_CONTEXT_ENVIRONMENT_PROD: 'design-system.post.ch,next.design-system.post.ch',
     STORYBOOK_GTM_PAGE_CONTEXT_ENVIRONMENT_FALLBACK: 'dev',
   }),
+  async viteFinal(config, options) {
+    return mergeConfig(config, {
+      css: {
+        devSourcemap: true,
+      },
+    });
```

Comment:
From this example: https://github.com/storybookjs/builder-vite#customize-vite-config